tests/unittests/CoreGraphics.drawing/CGContextDrawing_GradientTests.cpp

```diff
@@ -17,18 +17,18 @@
 #include "DrawingTest.h"
 
 #pragma region LinearGradient
-DRAW_TEST_F(CGGradient, LinearGradient, UIKitMimicTest) {
-    CGContextRef context = GetDrawingContext();
 
-    CGFloat locations[2] = { 0, 1 };
-    CGFloat components[8] = { 0.0, 0.0, 1, 1.0, 1.0, 0, 0, 1.0 };
+static void _drawLinearGradient(CGContextRef context,
+                                CGPoint startPoint,
+                                CGPoint endPoint,
+                                CGFloat* components,
```

Comment:
pass this in as an array rather than a pointer, and you can just use std::extent on it rather than passing in count